src/mmw/apps/bigcz/clients/cuahsi.py

```diff
@@ -45,16 +77,43 @@ def find_service(services, service_code):
     return None
 
 
-def parse_records(sites, services):
+def parse_records(series, services):
+    """
+    Join series catalog records to corresponding service.
+    """
     result = []
-    for site in sites:
-        service = find_service(services, site['servCode'])
+    for record in series:
+        service = find_service(services, record['ServCode'])
         if service:
-            record = parse_record(site, service)
+            record = parse_record(record, service)
             result.append(record)
     return result
 
 
+def group_series_by_location(series):
```

Comment:
`series` seems to be an overloaded term here. This takes `series`, and returns a grouping, but the values in the group are a set of key-value pairs, and not a `series`. Or is this common subset of fields across catalogs the true identifier of a series?